internal/databases/sqlserver/blob/server.go

```diff
@@ -72,6 +80,21 @@ func NewServer(folder storage.Folder) (*Server, error) {
 	bs.server = http.Server{Addr: bs.endpoint, Handler: bs}
 	bs.indexes = make(map[string]*Index)
 	bs.leases = make(map[string]Lease)
+	compressor, err := internal.ConfigureCompressor()
+	if err != nil {
+		if _, ok := err.(internal.UnknownCompressionMethodError); !ok {
```

Comment:
Maybe add some warning log to notify the user that WAL-G failed to configure the compressor?

Reply:
Agree.
Will return error unless **special** builtin compression method specified.